src/wpool_pool.erl

```diff
@@ -147,6 +148,19 @@ hash_worker(Sup, HashKey) ->
 cast_to_available_worker(Sup, Cast) ->
   wpool_queue_manager:cast_to_available_worker(queue_manager_name(Sup), Cast).
 
+%% @doc Casts a message to all the workers from the given pool.
+-spec cast_to_all_workers(wpool:name(), term()) -> ok.
+cast_to_all_workers(Sup, Cast) ->
+  QueueManager = queue_manager_name(Sup),
+  _ =
+    lists:foreach(
+      fun(Worker) ->
+        wpool_queue_manager:cast_to_worker(QueueManager, Cast, Worker)
```

Comment:
Why do you go through the queue when you could just use `wpool_process:cast(…)` directly?